### DISCUSSION TITLE
Request for Firebase App Indexing plugin

### DISCUSSION THREAD

**Discussion by kroikie** [firebase/flutterfire#5548] [Feature request]
We had an internal request to add this plugin. https://firebase.google.com/docs/app-indexing

It seems on iOS, this requires universal links. Do we have core support for this? It seems this is different from Firebase Dynamic Links :-/ 

For Android, the main plugin work is around enabling this: https://firebase.google.com/docs/app-indexing/android/personal-content

**Comment by kroikie** [comment]
@mehmetf

The issue at https://github.com/flutter/flutter/issues/35224 has been closed and moved here. Future collaboration on this issue will be done here.

**Comment by skol101** [comment]
+1

**Comment by Shankulkarni** [comment]
+1

**Comment by hsnbsst** [comment]
+1

**Comment by viveknimkarde** [comment]
+1

**Comment by viveknimkarde** [comment]
Any Updates  on this ?

**Comment by belisqui** [comment]
+1 It would be awesome to have this.

**Comment by palkesh-jain** [comment]
Much needed!

**Comment by hamad251** [comment]
+1

**Comment by DenicijaStojanovska** [comment]
+1

**Comment by marinat** [comment]
+1

**Comment by theniceboy** [comment]
+1

**Comment by marinat** [comment]
What workaround on android?  https://firebase.google.com/docs/app-indexing/android/personal-content this link doesn't work

**Comment by ohjs88** [comment]
+1

**Comment by fullflash** [comment]
+1

**Comment by jeffkelsey** [comment]
+1
